site/styles/_buttons.scss

```diff
@@ -244,3 +244,24 @@
     border-color: $gray-200;
   }
 }
+
+s72-donate-button:not(:empty) {
+  :hover {
```

Comment:
That should be `&:hover` for consistency.

Reply:
Hmmm a search of this file indicates  `@include hover-focus { ... }` as the pattern.

Reply:
Not saying one way is better than the other...

Reply:
So weirdly, this compiles to the following css (note the space:)
`s72-donate-button:not(:empty) :hover {`
Which means "hover on any child of s72-donate-button" aka `a:hover`

So you actually want this to be:
```
s72-donate-button:not(:empty) {
  &:hover a {
```

Reply:
Chur, thanks for the feedback. added &:hover a 👍